data/json/vehicleparts/vehicle_parts.json

```diff
@@ -2725,7 +2725,8 @@
     "difficulty": 4,
     "location": "under",
     "power": -300,
-    "flags": [ "PLOW", "EXTRA_DRAG" ],
+    "transform_terrain_data": { "pre_flags": [ "PLOWABLE" ], "post_terrain": "t_dirtmound", "post_field": "fd_fire", "post_field_intensity": 3, "post_field_age": "1 hour" },
```

Comment:
Plow will create fire fields with level 3 intensity while working?

Reply:
Just look how awesome fire plowing is:

![image](https://user-images.githubusercontent.com/16213433/60248017-6feb0600-98ca-11e9-864b-0347f9368e8e.png)

![image](https://user-images.githubusercontent.com/16213433/60248007-6a8dbb80-98ca-11e9-9004-63ce67863d09.png)

Reply:
The field, the field, the field is on fire.